code/game/gamemodes/machinegod/machinegod_items.dm

```diff
@@ -0,0 +1,427 @@
+/obj/item/clock_component
```

Comment:
i'll separate this shit further down the line, right now it's smashed together for my convenience more than anything